src/openzaak/templates/snippets/forms.html

```diff
@@ -54,3 +54,18 @@
     </div>
 </div>
 {% endblock %}
+
+
+{% block Toggle %}
+<div class="custom-control custom-switch">
+  <input name="{{ html_name }}" id="{{ id }}" type="checkbox" class="custom-control-input" {{ form_field.value|yesno:'checked,' }}>
```

Comment:
```suggestion
  <input name="{{ html_name }}" id="{{ id }}" type="checkbox" class="custom-control-input" {{ raw_value|yesno:'checked,' }}>
```
This is the typed value in sniplates, I think this should work.